app/src/main/java/org/simple/clinic/bp/history/BloodPressureHistoryScreen.kt

```diff
@@ -170,7 +169,7 @@ class BloodPressureHistoryScreen(
 
   private fun displayNameGenderAge(name: String, gender: Gender, age: Int) {
     val genderLetter = resources.getString(gender.displayLetterRes)
-    toolbar.title = resources.getString(R.string.bloodpressurehistory_toolbar_title, name, genderLetter, age)
+    toolbar.title = "$name, $genderLetter, $age"
```

Comment:
We should not be adding the comma in code. Reasons:

- There are languages which do not have commas
- There are languages where the comma glyph should be replaced by another character (The Ge'ez script used for Ethiopia languages has an alternate symbol that serves the same purpose as a comma), so we should use the correct glyph in those languages.